cmd/register/register.go

```diff
@@ -203,7 +203,7 @@ const (
 	defaultIpmiUser = "metal"
 
 	// defaultIpmiUserID the default id of the created ipmi user
-	defaultIpmiUserID = "10"
+	defaultIpmiUserID = "2"
```

Comment:
I would recommend to remove this once the lenovo test is done and move to go-hal

Reply:
Good point. We should also move the channel number and username